packages/edit-site/src/components/style-book/style.scss

```diff
@@ -0,0 +1,78 @@
+.edit-site-style-book {
+	background: $white; // Fallback color, overriden by JavaScript.
+	border-radius: $radius-block-ui;
+	bottom: 0;
+	left: 0;
+	overflow: hidden;
+	position: absolute;
+	right: 0;
+	top: 0;
+	transition: all 0.3s; // Match .block-editor-iframe__body transition.
+}
+
+.edit-site-style-book__close-button {
+	position: absolute;
+	right: $grid-unit-10;
+	top: ($grid-unit-60 - $button-size) / 2; // ( tab height - button size ) / 2
```

Comment:
Looks like this is generating a Sass deprecation warning for me locally. Would it be worth using a `calc()` rule here?

<img width="1067" alt="image" src="https://user-images.githubusercontent.com/14988353/205525855-32bfece2-d6e0-40fc-9259-d88202511f4a.png">

Reply:
Oh! Didn't know this was deprecated. Looks like `math.div` is appropriate here.

https://sass-lang.com/documentation/breaking-changes/slash-div

Reply:
Nice one, TIL about `math.div` 😄

Reply:
Look at us learning stuff.

Reply:
![image](https://user-images.githubusercontent.com/6458278/205526621-5daaa14f-e986-48c5-af3a-6eab55f663c6.png)